docs/product/alerts/alert-types.mdx

```diff
@@ -37,6 +37,10 @@ The **Alert Details** page also includes a list of issues that triggered the ale
 
 Metric alerts tell you when a [metric](/product/performance/metrics/) crosses a threshold set by you, like a spike in the number of errors in a project, or a change in a performance metric, such as [transaction duration](/product/performance/metrics/#latency), [Apdex](/product/performance/metrics/#apdex), [failure rate](/product/performance/metrics/#failure-rate), or [throughput](/product/performance/metrics/#throughput-total-tpm-tps).
 
+<Note>
+You can use [dynamic alerts](/product/alerts/create-alerts/metric-alert-config/#dynamic-alerts) to let Sentry define the threshold for you.
+</Note>
+
```

Comment:
```suggestion
You can use [dynamic alerts](/product/alerts/create-alerts/metric-alert-config/#dynamic-alerts) to let Sentry define the threshold for you.

```
I'd take this out of the note format and just add it to the paragraph above. There are a lot of different notes in this section which makes reading a bit jarring.